.github/workflows/diff_report.yml

```diff
@@ -45,14 +45,20 @@ jobs:
        id: parse
        run: |
         grep "^Diff Regression projects:" text > temp || echo "" > temp
-        sed 's/Diff Regression projects: //' temp > project
-        echo ::set-output name=project_link::$(cat project)
+        sed 's/Diff Regression projects: //' temp > projects_link
+        echo ::set-output name=projects_link::$(cat projects_link)
+
         grep "^Diff Regression config:" text > temp || echo "" > temp
-        sed 's/Diff Regression config: //' temp > config
-        echo ::set-output name=config_link::$(cat config)
+        sed 's/Diff Regression config: //' temp > config_link
+        echo ::set-output name=config_link::$(cat config_link)
+
+        grep "^New module config:" text > temp || echo "" > temp
```

Comment:
`grep "^New module config:" text | cat > temp` might be better form

Reply:
done